datascience/tables.py

```diff
@@ -98,6 +86,14 @@ def __init__(self, columns=None, labels=None,
             self[label] = column
 
     @classmethod
+    def empty(cls, column_labels=None):
+        """Create an empty table."""
```

Comment:
Let's document this method a little better; just describing the args and return value would be very helpful for someone wondering how to use it.